src/loaders/external_jpg/tvgJpgLoader.h

```diff
@@ -48,6 +50,7 @@ class JpgLoader : public LoadModule
     unsigned char *image = nullptr;
     unsigned long size = 0;
     bool freeData = false;
+    uint32_t tjpfColorSpace = TJPF_BGRX;
```

Comment:
I think a new variable is not necessary because Loader already has a colorSpace variable.
you can add a local converter function and immediately call it when it's necessary (see mentioned the above comment)